systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -151,4 +153,47 @@ public static void waitForClusterStability(String clusterName) {
             return false;
         });
     }
+
+    /**
+     * Method which, update/replace Kafka configuration
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     */
+    public static void updateSpecificConfiguration(String clusterName, KafkaDynamicConfiguration kafkaDynamicConfiguration, Object value) {
+        KafkaResource.replaceKafkaResource(clusterName, kafka -> {
+            LOGGER.info("Kafka config before updating '{}'", kafka.getSpec().getKafka().getConfig().toString());
+            Map<String, Object> config = kafka.getSpec().getKafka().getConfig();
+            config.put(kafkaDynamicConfiguration.toString(), value);
+            kafka.getSpec().getKafka().setConfig(config);
+            LOGGER.info("Kafka config after updating '{}'", kafka.getSpec().getKafka().getConfig().toString());
+        });
+    }
+
+    /**
+     * Method which, extends the @see updateConfiguration(String clusterName, KafkaConfiguration kafkaConfiguration, Object value) method
+     * with stability and ensures after update of Kafka resource there will be not rolling update
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     */
+    public static void updateConfigurationWithStabilityWait(String clusterName, KafkaDynamicConfiguration kafkaDynamicConfiguration, Object value) {
+        updateSpecificConfiguration(clusterName, kafkaDynamicConfiguration, value);
+        PodUtils.verifyThatRunningPodsAreStable(KafkaResources.kafkaStatefulSetName(clusterName));
+    }
+
+    /**
+     * Method, which encapsulates the update phase of dyn. configuration + verifying that updating configuration were successfully done
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     */
+    public static void verifyDynamicConfiguration(String clusterName, KafkaDynamicConfiguration kafkaDynamicConfiguration, Object value) {
+        KafkaUtils.updateConfigurationWithStabilityWait(clusterName, kafkaDynamicConfiguration, value);
+
+        boolean result = KafkaResource.kafkaClient().inNamespace(kubeClient().getNamespace()).withName(clusterName).get().getSpec().getKafka().getConfig().get(kafkaDynamicConfiguration.toString()) == value;
```

Comment:
Wouldn't it be better to make some variable for this? Because you use it in `AssertionError` too.

Reply:
It was not a good approach on how to verify I have changed the it....